src/seedu/addressbook/data/person/Contact.java

```diff
@@ -0,0 +1,11 @@
+package seedu.addressbook.data.person;
+
+public class Contact {
+
+    protected boolean isPrivate;
+    
```

Comment:
There are more variables and methods that can be reduced and placed in the Contact class (parent) as well.